service/src/main/java/org/cbioportal/genome_nexus/service/EnsemblService.java

```diff
@@ -16,4 +16,6 @@ EnsemblTranscript getCanonicalEnsemblTranscriptByHugoSymbol(String hugoSymbol, S
 
     List<EnsemblTranscript> getEnsemblTranscripts(String geneId, String proteinId, String hugoSymbol);
     List<EnsemblTranscript> getEnsemblTranscripts(List<String> transcriptIds, List<String> geneIds, List<String> proteinIds, List<String> hugoSymbols);
+
+    String getEnsemblGene(String hugoSymbol);
```

Comment:
Gentlemen, please implement your methods :)

`[ERROR] /home/travis/build/genome-nexus/genome-nexus/service/src/main/java/org/cbioportal/genome_nexus/service/internal/EnsemblServiceImpl.java:[15,8] org.cbioportal.genome_nexus.service.internal.EnsemblServiceImpl is not abstract and does not override abstract method getEnsemblGene(java.lang.String) in org.cbioportal.genome_nexus.service.EnsemblService`